src/components/router/router.tsx

```diff
@@ -5,21 +5,22 @@ import { LeaderboardPage } from 'pages/leaderboard';
 import { SignInPage } from 'pages/signin';
 import { SignUpPage } from 'pages/signup';
 import { StartGamePage } from 'pages/start-game';
-import { UserPage } from 'pages/user';
 import { WelcomePage } from 'pages/welcome-page';
-import { Route, Routes } from 'react-router-dom';
-
-const user = false;
+import { useSelector } from 'react-redux';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { isLoggedInIfoSelector } from 'store/auth-reducer';
 
 export const AppRouter = () => {
-  if (user) {
+  const authState = useSelector(isLoggedInIfoSelector);
+
+  if (authState.isLoggedIn) {
     return (
       <MainLayout>
         <Routes>
           <Route path="/game" element={<StartGamePage />} />
-          <Route path="/user" element={<UserPage />} />
           <Route path="/leaderboard" element={<LeaderboardPage />} />
           <Route path="/forum" element={<ForumPage />} />
+          <Route path="*" element={<Navigate to="/game" />} />
```

Comment:
тут типа должна быть 404?

если это ее замена, почему вот так не сделал?
```suggestion
          <Route path="*" element={<StartGamePage />} />
```

Reply:
Поменял на `<Route path="*" element={<StartGamePage />} />`